apps/pkeyutl.c

```diff
@@ -39,6 +42,14 @@ static int do_raw_keyop(int pkey_op, EVP_MD_CTX *mctx,
                         int filesize, unsigned char *sig, int siglen,
                         unsigned char **out, size_t *poutlen);
 
+static int only_rawin(const EVP_PKEY *pkey)
+{
+    if (pkey == NULL)
+        return 0;
+    return EVP_PKEY_get_id(pkey) == EVP_PKEY_ED25519
+        || EVP_PKEY_get_id(pkey) == EVP_PKEY_ED448;
+}
+
```

Comment:
Shouldn't it be `_is_a` instead ?

Reply:
Can do - which will be the right string argument?

Reply:
Hmm, I found one cannot use `EDDSA` or the like.
Would this be any better?
```
    return EVP_PKEY_is_a(pkey, "ED25519")
        || EVP_PKEY_is_a(pkey, "ED448");
```
Likely it's less efficient.

Reply:
I wouldn't worry about effectiveness here as the command-line util is not intended to be very effective

Reply:
Right, efficiency does not really matter here.
So changed it this way.

Reply:
Thanks!